operatorcourier/validate.py

```diff
@@ -208,43 +216,48 @@ def _pkgs_validation(self, bundleData):
 
         pkgs = bundleData[self.pkgsKey]
 
-        for pkg in pkgs:
-            if "packageName" in pkg:
-                logger.info("Evaluating package %s", pkg["packageName"])
-            else:
-                logger.error("packageName not defined.")
-                valid = False
+        if len(pkgs) > 1:
+            logger.error('Only 1 package is expected to exist per bundle.')
+            valid = False
 
-            if "channels" in pkg:
-                channels = pkg["channels"]
-                if len(channels) == 0:
-                    logger.error("no package channels defined.")
-                    valid = False
-                else:
-                    csvNames = []
-                    for csv in bundleData[self.csvKey]:
-                        try:
-                            csvName = csv["metadata"]["name"]
-                            csvNames.append(csvName)
-                        except KeyError:
-                            pass
-                    for channel in channels:
-                        if "name" not in channel:
-                            logger.error("package channel.name not defined.")
-                            valid = False
+        pkg = pkgs[0]
```

Comment:
What about the case where pkgs == 0 ?

Reply:
@kevinrizza It should already be handled [here](https://github.com/operator-framework/operator-courier/blob/master/operatorcourier/validate.py#L254) before the pkg validator is called.

Reply:
@jeremylinlin we can't rely on other validation methods. what if they were reordered or that other method was refactored?

Reply:
@kevinrizza Ah that makes sense. I'll fix it. Thanks Kevin!